docs/web3.eth.rst

```diff
@@ -987,6 +987,45 @@ The following methods are available on the ``web3.eth`` namespace.
     As of v6.3.0, the raw data is also returned and
     can be accessed via the ``data`` attribute on ``ContractLogicError``.
 
+
+.. py:method:: Eth.create_access_list(transaction, block_identifier=web3.eth.default_block)
+
+    * Delegates to ``eth_createAccessList`` RPC Method
+
+    This method creates an `EIP-2930 <https://eips.ethereum.org/EIPS/eip-2930>`_ type ``accessList`` based on a given
+    ``Transaction``. The ``accessList`` contains all storage slots and addresses read and written by the transaction,
+    except for the sender account and the precompiles. This method uses the same ``transaction`` call object and
+    ``block_identifier`` object as :meth:`~web3.eth.Eth.call()`. An ``accessList`` can be used to unstick contracts that
+    became inaccessible due to gas cost increases.
+
+    :param transaction: ``TransactionCall`` object.
```

Comment:
Is a `TransactionCall` object defined somewhere? I'm not finding anything in our docs, and geth docs only have broken links - https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-eth#eth-createaccesslist

Reply:
I didnt see that, must be something old or they made it up. I assume this is supposed to point to a class, but maybe the type `TxParams` is appropriate?

Reply:
Yeah let's change this up. For `Eth.call` we say: 

"The ``transaction`` parameter is handled in the same manner as the :meth:`~web3.eth.Eth.send_transaction()` method."

Perhaps that's good enough here too since we go into depth there on each param. Thoughts?